board/opendingux/Config.in

```diff
@@ -23,6 +23,7 @@ comment "Libraries"
 	source "$BR2_EXTERNAL_OPENDINGUX_PATH/package/libxdgmime/Config.in"
 	source "$BR2_EXTERNAL_OPENDINGUX_PATH/package/love2d/Config.in"
 	source "$BR2_EXTERNAL_OPENDINGUX_PATH/package/utf8cpp/Config.in"
+	source "$BR2_EXTERNAL_OPENDINGUX_PATH/package/bennugd/Config.in"
```

Comment:
Please sign this commit with `git commit --amend -s` so that it can be applied upstream as well (https://github.com/OpenDingux/buildroot)